### DISCUSSION TITLE
Nvidia 4060ti ventus 3x 8gb OC - Fan control not working

### DISCUSSION THREAD

**Discussion by Lautaro1109** [Rem0o/FanControl.Releases#2485] [Q&A]
I apologize to everyone because my english is not good, but anyways im gonna try to explain what is happening.

I have a problem trying to modify my GPU FAN 1. 

![image](https://github.com/Rem0o/FanControl.Releases/assets/66539041/af4c20d3-d64c-4404-9d71-a262cea0fafd)
 
 It says that this control has no RPM sensor. I tried everything. A couple months ago i could use it but idk why i cant do it now.

Like i said, i tried everything. I changed my PCIe cable, i updated my bios, y unpluged the gpu and i connect it again. I uninstall all the drivers and i restored my windows to default. But i still cant manage my cpu fan 1. And it is running at 100% when i boot the pc.

![image](https://github.com/Rem0o/FanControl.Releases/assets/66539041/b9dd76d3-f0b1-4266-ac96-28aa1bde0e22)

PC SPECS: 

NVIDIA 4060 TI VENTUS 3X 8GB OC
MOTHERBOARD ASROCK B450 STEEL LEGEND
16 GB RAM DDR4 CORSAIR VENGEANCE - X2
PROCESSOR AMD RYZEN 9 5900X 12-CORE 3.70 GHz
PSU COOLER MASTER MWE Gold 750 - V2

**Comment by Rem0o** [comment]
Does it respond to afterburner?

**Reply by extinctleads** [reply]
I've ran into a similar issue with these latest updates. GPU fans (pump actually) and GPU temps worked fine for 5+ years. Today, they stopped working properly, and no amount of reinstalling, calibrating, NVIDIA drivers, etc., work. FanControl can control the GPU fan/pump speed, but it shows as 0 RPM, and I keep getting sensor errors. I also can't get a GPU temperature reading. Strangely, CPU pump is hovering around 800 RPM even when manually set to 0%.

Although Afterburner at least **_detects_** GPU temp, it doesn't seem to be able to control the fan (or pump in my case).

EDIT: To be clear -- FanControl **_does_** control the pump speed. However, it shows at 0 RPM even when manually set to 100%. Also, CPU fan somehow is hovering 800 RPM when set to 0%?



![image](https://github.com/Rem0o/FanControl.Releases/assets/168778023/da8e917f-04cd-4534-b68b-5921dbb39f4a)

**Reply by Lautaro1109** [reply]
It doesnt respond to afterburner too.

**Reply by Rem0o** [reply]
@Lautaro1109  just to be clear, only one set of fans (those 3 fans card always have 1 group of 2 responding together, and there's the other fan alone) respond?   Of course make sure to have the latest Nvidia drivers installed, no need for Geforce experience by the way, then put both the manual control sliders on the control cards for your GPU to 50, have the RPM visibles, and send a screenshot of that. Also, save that config (json file), and send it here.

**Reply by Lautaro1109** [reply]
Yeah, like u said, i have only 1 set responding to fan control. It is the set that has 2 fans. And the another set of 1 fan its not responding.

I have done the hole proccess again. And that is that i have for u. These screenshots refer to the steps.

1- Sensor settings
![Screen 1](https://github.com/Rem0o/FanControl.Releases/assets/66539041/4ef92be0-1196-419e-8154-dfe1a0b2f9c0)

2- Speed calibration, as u can see, control 1 of nvidia said that "this control has no RPM sensor"
![Screen 2](https://github.com/Rem0o/FanControl.Releases/assets/66539041/2be802a0-7c34-46b1-bdf1-783c243e5d77)

3- Only fan 2 of cpu and set fan 2 of gpu are calibrated.
![Screen 3](https://github.com/Rem0o/FanControl.Releases/assets/66539041/7a0e7cf2-cc6a-4471-84c6-14cd2da5d84f)

4- Then i had to calibrate manually the control 1 of gpu
![Screen 4](https://github.com/Rem0o/FanControl.Releases/assets/66539041/f9acf1af-bbf9-4075-ab40-0ad764de88d1)

5- And finally i have both calibrated but control 1 keep without response.
![Screen 5](https://github.com/Rem0o/FanControl.Releases/assets/66539041/9fb99b1d-e3fd-4bda-b684-cc7864cd5974)



JSON CONFIG.

{
  "__VERSION__": "188",
  "Main": {
    "Controls": [
      {
        "Calibration": [],
        "Enable": false,
        "ForceApply": false,
        "Identifier": "/lpc/nct6779d/control/0",
        "IsHidden": true,
        "ManualControl": false,
        "ManualControlValue": 50,
        "MinimumPercent": 0,
        "Name": "Fan #1",
        "NickName": "Fan #1",
        "PairedFanSensor": null,
        "SelectedCommandStepDown": 8.0,
        "SelectedCommandStepUp": 8.0,
        "SelectedFanCurve": null,
        "SelectedOffset": 0,
        "SelectedStart": 0,
        "SelectedStop": 0
      },
      {
        "Calibration": [
          [
            10,
            423
          ],
          [
            20,
            456
          ],
          [
            30,
            630
          ],
          [
            40,
            806
          ],
          [
            50,
            954
          ],
          [
            60,
            1096
          ],
          [
            70,
            1222
          ],
          [
            80,
            1348
          ],
          [
            90,
            1461
          ],
          [
            100,
            1619
          ]
        ],
        "Enable": false,
        "ForceApply": false,
        "Identifier": "/lpc/nct6779d/control/1",
        "IsHidden": false,
        "ManualControl": false,
        "ManualControlValue": 50,
        "MinimumPercent": 0,
        "Name": "Fan #2",
        "NickName": "Fan #2",
        "PairedFanSensor": {
          "Identifier": "/lpc/nct6779d/fan/1",
          "IsHidden": false,
          "Name": "Fan #2",
          "NickName": "Fan #2"
        },
        "SelectedCommandStepDown": 8.0,
        "SelectedCommandStepUp": 8.0,
        "SelectedFanCurve": null,
        "SelectedOffset": 0,
        "SelectedStart": 0,
        "SelectedStop": 0
      },
      {
        "Calibration": [],
        "Enable": false,
        "ForceApply": false,
        "Identifier": "/lpc/nct6779d/control/2",
        "IsHidden": true,
        "ManualControl": false,
        "ManualControlValue": 50,
        "MinimumPercent": 0,
        "Name": "Fan #3",
        "NickName": "Fan #3",
        "PairedFanSensor": null,
        "SelectedCommandStepDown": 8.0,
        "SelectedCommandStepUp": 8.0,
        "SelectedFanCurve": null,
        "SelectedOffset": 0,
        "SelectedStart": 0,
        "SelectedStop": 0
      },
      {
        "Calibration": [],
        "Enable": false,
        "ForceApply": false,
        "Identifier": "/lpc/nct6779d/control/3",
        "IsHidden": true,
        "ManualControl": false,
        "ManualControlValue": 50,
        "MinimumPercent": 0,
        "Name": "Fan #4",
        "NickName": "Fan #4",
        "PairedFanSensor": null,
        "SelectedCommandStepDown": 8.0,
        "SelectedCommandStepUp": 8.0,
        "SelectedFanCurve": null,
        "SelectedOffset": 0,
        "SelectedStart": 0,
        "SelectedStop": 0
      },
      {
        "Calibration": [],
        "Enable": false,
        "ForceApply": false,
        "Identifier": "/lpc/nct6779d/control/4",
        "IsHidden": true,
        "ManualControl": false,
        "ManualControlValue": 50,
        "MinimumPercent": 0,
        "Name": "Fan #5",
        "NickName": "Fan #5",
        "PairedFanSensor": null,
        "SelectedCommandStepDown": 8.0,
        "SelectedCommandStepUp": 8.0,
        "SelectedFanCurve": null,
        "SelectedOffset": 0,
        "SelectedStart": 0,
        "SelectedStop": 0
      },
      {
        "Calibration": [
          [
            0,
            1200
          ],
          [
            10,
            1500
          ],
          [
            40,
            2500
          ],
          [
            70,
            3000
          ],
          [
            100,
            3600
          ]
        ],
        "Enable": true,
        "ForceApply": true,
        "Identifier": "NVApiWrapper/0-AD106-A/control/0",
        "IsHidden": false,
        "ManualControl": true,
        "ManualControlValue": 50,
        "MinimumPercent": 0,
        "Name": "Control 1 - NVIDIA GeForce RTX 4060 Ti",
        "NickName": "Control 1 - NVIDIA GeForce RTX 4060 Ti",
        "PairedFanSensor": {
          "Identifier": "NVApiWrapper/0-AD106-A/fan/0",
          "IsHidden": false,
          "Name": "Fan 1 - NVIDIA GeForce RTX 4060 Ti",
          "NickName": "Fan 1 - NVIDIA GeForce RTX 4060 Ti"
        },
        "SelectedCommandStepDown": 8.0,
        "SelectedCommandStepUp": 8.0,
        "SelectedFanCurve": null,
        "SelectedOffset": 0,
        "SelectedStart": 0,
        "SelectedStop": 0
      },
      {
        "Calibration": [
          [
            0,
            0
          ],
          [
            10,
            1203
          ],
          [
            20,
            1204
          ],
          [
            30,
            1207
          ],
          [
            40,
            1462
          ],
          [
            50,
            1814
          ],
          [
            60,
            2184
          ],
          [
            70,
            2541
          ],
          [
            80,
            2913
          ],
          [
            90,
            3254
          ],
          [
            100,
            3529
          ]
        ],
        "Enable": true,
        "ForceApply": true,
        "Identifier": "NVApiWrapper/0-AD106-A/control/1",
        "IsHidden": false,
        "ManualControl": true,
        "ManualControlValue": 50,
        "MinimumPercent": 0,
        "Name": "Control 2 - NVIDIA GeForce RTX 4060 Ti",
        "NickName": "Control 2 - NVIDIA GeForce RTX 4060 Ti",
        "PairedFanSensor": {
          "Identifier": "NVApiWrapper/0-AD106-A/fan/1",
          "IsHidden": false,
          "Name": "Fan 2 - NVIDIA GeForce RTX 4060 Ti",
          "NickName": "Control 2 - NVIDIA GeForce RTX 4060 Ti"
        },
        "SelectedCommandStepDown": 8.0,
        "SelectedCommandStepUp": 8.0,
        "SelectedFanCurve": null,
        "SelectedOffset": 0,
        "SelectedStart": 9,
        "SelectedStop": 6
      }
    ],
    "CustomSensors": [],
    "FanCurves": [
      {
        "CommandMode": 0,
        "IsHidden": false,
        "Name": "Flat curve example",
        "Percent": 50.0
      }
    ],
    "FanSensors": [
      {
        "Identifier": "/lpc/nct6779d/fan/0",
        "IsHidden": false,
        "Name": "Fan #1",
        "NickName": "Fan #1"
      },
      {
        "Identifier": "/lpc/nct6779d/fan/1",
        "IsHidden": false,
        "Name": "Fan #2",
        "NickName": "Fan #2"
      },
      {
        "Identifier": "/lpc/nct6779d/fan/2",
        "IsHidden": false,
        "Name": "Fan #3",
        "NickName": "Fan #3"
      },
      {
        "Identifier": "/lpc/nct6779d/fan/3",
        "IsHidden": false,
        "Name": "Fan #4",
        "NickName": "Fan #4"
      },
      {
        "Identifier": "/lpc/nct6779d/fan/4",
        "IsHidden": false,
        "Name": "Fan #5",
        "NickName": "Fan #5"
      },
      {
        "Identifier": "NVApiWrapper/0-AD106-A/fan/0",
        "IsHidden": false,
        "Name": "Fan 1 - NVIDIA GeForce RTX 4060 Ti",
        "NickName": "Fan 1 - NVIDIA GeForce RTX 4060 Ti"
      },
      {
        "Identifier": "NVApiWrapper/0-AD106-A/fan/1",
        "IsHidden": false,
        "Name": "Fan 2 - NVIDIA GeForce RTX 4060 Ti",
        "NickName": "Control 2 - NVIDIA GeForce RTX 4060 Ti"
      }
    ],
    "HideCalibration": false,
    "HideFanSpeedCards": false,
    "HorizontalUIOrientation": false,
    "PrimaryColor": "#FF607D8B",
    "SecondaryColor": "#FFC6FF00",
    "SelectedTheme": "Inherit",
    "ShowHiddenCards": true,
    "SyncThemeWithWindows": true,
    "SyncTrayIconColorWithWindows": true,
    "TemperatureSensors": [],
    "TrayIconColor": "#FFFFFFFF",
    "TrayIcons": []
  },
  "Sensors": {
    "DisabledPlugins": [],
    "DisableStorageSensors": true,
    "LibreHardwareMonitorSettings": {
      "Controller": true,
      "CPU": true,
      "EmbeddedEC": true,
      "GPU": true,
      "InpOut": true,
      "Memory": false,
      "Motherboard": true,
      "PSU": false,
      "Storage": false,
      "ZeroRPMOverride": true
    },
    "NvAPIWrapperSettings": {
      "Enabled": true,
      "ZeroRPMOverride": true
    },
    "SensorCount": 26
  }
}

[userConfig.json](https://github.com/Rem0o/FanControl.Releases/files/15201235/userConfig.json)

**Reply by Rem0o** [reply]
From what I can see, everything is in order.  The fact it doesn't respond at all and runs max speed all the time, most likely your GPU has a defect.

**Reply by Lautaro1109** [reply]
Ok, thank u so much rem0o. U were really helpfull to me.
